cron/src/main/scala/com/criteo/cuttle/cron/package.scala

```diff
@@ -30,6 +30,10 @@ package object cron {
     }
 
     implicit val cronContextShift = IO.contextShift(cronThreadPool.underlying)
+
+    //For Cron Sleep
+    implicit val timer =
```

Comment:
There is already a threadPool for waiting in the Timeout object

Reply:
Done